homeassistant/components/somfy/__init__.py

```diff
@@ -95,7 +94,10 @@ async def _update_all_devices():
         previous_devices = data[COORDINATOR].data
         # Sometimes Somfy returns an empty list.
         if not devices and previous_devices:
-            raise UpdateFailed("No devices returned")
+            _LOGGER.warning(
```

Comment:
```suggestion
            _LOGGER.debug(
```

This will generate issue reports for an expected condition if we set it at warning